core-ui/src/components/Predefined/Create/ConfigMaps/ConfigMaps.create.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+import { ConfigMapForm } from './ConfigMapForm';
+
+export function ConfigMapsCreate(props) {
```

Comment:
This file seems unnecessary. Maybe move the content of ConfigMapForm.js to ConfigMaps.create.js